docs/20_smart-contracts/10_getting-started/10_dune-guide.md

```diff
@@ -2,55 +2,55 @@
 title: DUNE
 ---
 
-[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool that allows blockchain developers and node operators to perform boilerplate tasks related to smart contract development and node management functions.
+[Docker Utilities for Node Execution (DUNE)](https://github.com/AntelopeIO/DUNE) is a client tool. 
+As a blockchain developer or node operator, use DUNE for your common tasks.
 
-Before getting started with smart contract development, you need to learn about DUNE and how to install it on your platform.
+Before you begin, learn about DUNE and how to install it on your platform.
 
 ### Installation
 
-DUNE can be installed and run on the following platforms:
+You can install and run DUNE on the following platforms:
 * Linux
 * Windows
 * MacOS
 
-Installation instructions for each supported platform are available on the [DUNE's github project](https://github.com/AntelopeIO/DUNE) page.
+Each platform has their own installation instructions. View the [DUNE GitHub](https://github.com/AntelopeIO/DUNE) for installation instructions. 
 
-Once you're done, you can run `dune --help` to see a list of all supported commands.
+Run `dune --help` to view a list of all supported commands.
 
 ## Wallets
 
-DUNE handles wallet management for you so that you don't have to. 
+DUNE handles wallet management for you so you don't have to. 
 
 If you need to import a new key into your wallet:
 
 ```shell
 dune --import-dev-key <PRIVATE_KEY>
 ```
 
-## Node Management
+## Node management
 
-Creating a new local EOS blockchain is easy with DUNE.
+Use DUNE to easily create a new local EOS blockchain.
+
+The above command creates a new node called `NODE_NAME`.  This node initiates with default settings. These default settings configure the node to serve as an API/producer node. You can deploy smart contracts to it and perform tests on it.
```

Comment:
Please revert the changes here as per this comment: https://github.com/eosnetworkfoundation/docs/pull/118#discussion_r1192244820

This now says the command above for the command below.